src/main/java/com/onionshop/controllers/ProjectStateController.java

```diff
@@ -152,11 +144,12 @@ else if (colourButton.getButton() == MouseButton.SECONDARY) {
     protected void onCanvasMouseDragged(MouseEvent canvasMouseLocation) {
         int[][] updatedPixels = canvasInputProcessor.processControllerDataForDrawingManager(canvasMouseLocation);
 
-
         PixelWriter canvasPixelWriter = projectDrawing.getGraphicsContext2D().getPixelWriter();
         for (int i = 0; i < updatedPixels.length; i++) {
             canvasPixelWriter.setColor(updatedPixels[i][0], updatedPixels[i][1], currentCanvasColour);
         }
+
+        System.out.println(ToolStateManager.getInstance().getCurrentToolState());
```

Comment:
Is this necessary? Remember to get rid of this.

Reply:
whoops no its not I tots forgot